.coafile

```diff
@@ -4,6 +4,10 @@ bears = SpaceConsistencyBear, LineLengthBear, CoffeeLintBear
 use_spaces = True
 max_line_length = 80
 
+[PYTHON]
+files = **.py
+bears = PEP8Bear
+
```

Comment:
Done. It shows up on the next line because of the review on a blank line.
We do have a blank line before default_actions, so it should be alright like this.